packages/oid4vci-demo-frontend/src/configs/sphereon.json

```diff
@@ -86,6 +86,20 @@
         }
       }
     },
+    "credSelect": {
```

Comment:
This is not really a good name. Suggest to use something like multiCedentialIssuer or something

Reply:
this is not added by this ticket and needs a refactor for all the configuration files. making a ticket for this

Reply:
Well, this ticket is about now having a single credential issue screen and a multi credential issue screen, so I would reckon this is the right PR to make this disctinction in the name clear

Reply:
changed it to `multiCedentialIssuer` in here and everywhere that we're using it